editor/components/post-author/check.js

```diff
@@ -11,17 +11,23 @@ import { compose } from '@wordpress/element';
 
 export function PostAuthorCheck( { user, users, children } ) {
 	const authors = filter( users.data, ( { capabilities } ) => capabilities.level_1 );
-	if ( ! user.data || ! user.data.capabilities.publish_posts || authors.length < 2 ) {
+	const userCaps = user.data ?
+		{ ...user.data.capabilities, ...user.data.post_type_capabilities } :
+		{ 'publish_posts': false };
+
+	if ( ! userCaps.publish_posts || authors.length < 2 ) {
 		return null;
 	}
 
 	return children;
 }
 
 const applyWithAPIData = withAPIData( () => {
+	const postTypeSlug = window._wpGutenbergPost.type;
```

Comment:
We should avoid using the global here, and rely on the `getCurrentPostType` selector. Example here 
https://github.com/WordPress/gutenberg/blob/master/editor/components/page-attributes/check.js#L32-L51

Reply:
Thanks for your feedback. I've tried to use `getCurrentPostType` but unfortunately the Post Type was not defined yet at the time the request was sent. I'll give it another look asap.